src/pages/instances/Events.tsx

```diff
@@ -1,8 +1,14 @@
 import React, { FC, useEffect, useState } from "react";
 import { LxdEvent } from "types/event";
 import { useEventQueue } from "context/eventQueue";
+import { useAuth } from "context/auth";
 
 const Events: FC = () => {
+  const { isAuthenticated } = useAuth();
+  if (!isAuthenticated) {
+    return null;
+  }
```

Comment:
This is breaking the rules of hooks:
```
Warning: React has detected a change in the order of Hooks called by Events. This will lead to bugs and errors if not fixed. For more information, read the Rules of Hooks: https://reactjs.org/link/rules-of-hooks

   Previous render            Next render
   ------------------------------------------------------
1. useContext                 useContext
2. undefined                  useContext
   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    at Events (https://localhost:8407/src/pages/instances/Events.tsx:24:7)
```

Reply:
True this changes when a user logs in, I'll change this slightly, so it is conforming.